control-board/src/tasks/control_task.rs

```diff
@@ -320,8 +359,16 @@ impl <
                 // defmt::info!("wheel curr: {} {} {} {}", self.motor_fl.read_current(), self.motor_bl.read_current(), self.motor_br.read_current(), self.motor_fr.read_current());
 
 
-                self.send_motor_commands_and_telemetry(
-                    &mut robot_controller, cur_state, self.last_battery_v);
+                ///////////////////////////////////
+                //  send commands and telemetry  //
+                ///////////////////////////////////
+
+                self.send_motor_commands_and_telemetry(seq_number,
+                    &mut robot_controller, cur_state);
+
+                // increment seq number
+                seq_number += 1;
+                seq_number &= 0xFFF;
```

Comment:
Is there a reason this is masked to 12 bits?

Reply:
I think underlying data type is uint16_t, and rust panics on overflow by default, so I just picked an easy and sufficiently large number.